### ISSUE TITLE
Feature request: Windows user_home environmet variable

### ISSUE THREAD

**Issue by PavelSosin-320** [microsoft/WSL#4776]
*can't access DockerDesktop Kubernetes instance from Ubuntu Kubernetes client utilities.**
A clear and concise description of what the problem is. Ex. I'm always frustrated when [using helm or kubctl from Ubuntu windows they can't connect]
KUBECONFIG env variable is normally expressed via UserHome as $User_home/.kube/ etc.
**Desired solution:**
Inject Windows_user_home =/mnt/c/User<UsersName ......... environment variable which translates and map Windows User_home to Linux ~ to facilitate access to various .xxx folders containing per-user configuration files which are invariant for Windows and Linux like kubeconfig, , mvn, etc.

**Describe alternatives you've considered**

**Additional context**

**Comment by benhillis** [comment]
WSL has the ability to share environment variables: https://devblogs.microsoft.com/commandline/share-environment-vars-between-wsl-and-windows/

For example you can accomplish what you're asking for by setting up `%WSLENV%` as such:
```
C:\Users\benhill>set WSLENV=USERPROFILE/up

C:\Users\benhill>wsl
root@BENHILL-DELL:/c/Users/benhill# echo $USERPROFILE
/c/Users/benhill
root@BENHILL-DELL:/c/Users/benhill#
```

**Comment by PavelSosin-320** [comment]
Off course almost every unix originated utility has its own way to provide configuration file location and I achieved desired kubctl behavior in my Ubuntu distro for my local Docker Desktop cluster in the following way:

_$ kubectl --kubeconfig /mnt/c/Users/Pavel/.kube/config cluster-info
Kubernetes master is running at https://kubernetes.docker.internal:6443
KubeDNS is running at https://kubernetes.docker.internal:6443/api/v1/namespaces/kube-system/services/kube-dns:dns/proxy_

To further debug and diagnose cluster problems, use 'kubectl cluster-info dump'._

But it took me as long as my morining coffee to install all Cloud/Azure/Kubernetes essential client side utilities using apt-get. I spent 2 days to install and config Helm 3 for Windows.
operator-sdk even has no official Windows release yet. Ansimble Galaxy contains bulk of Azure operators.
